lib/docker-sync/dependencies/package_manager.rb

```diff
@@ -0,0 +1,18 @@
+module DockerSync
+  module Dependencies
+    module PackageManager
+      NO_SUPPORTED_PACKAGE_MANAGER = 'No package manager available for your OS. Please file an issue on https://github.com/eugenmayer/docker-sync/issues mentioning your OS and favourite package manager.'.freeze
+
+      class << self
+        extend Forwardable
+        def_delegators :package_manager, :available?, :ensure!, :install_package
+      end
+
+      def self.package_manager
+        return @package_manager if defined? @package_manager
+        return @package_manager = PackageManager::Brew if Environment.mac?
+        raise(NO_SUPPORTED_PACKAGE_MANAGER)
```

Comment:
Is this requiring them to have brew installed even if they are using `native_osx` where it should not be needed?